cookbook/product/manipulate-product.rst

```diff
@@ -0,0 +1,12 @@
+How to Manipulate the Products
+==============================
+
+This chapter gives details on how to programmatically manipulate products, the product is the main object of the PIM and it comes with specific services for the following uses.
+
+.. toctree::
+
+   query-products
+   create-product
```

Comment:
will change all urls, would love to do but i need to do some apache rewrite to do it in a SEO friendly ay, i think we'll re-organize this before the 1.4 stable